karma.conf.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function(config) {
```

Comment:
It's missing the webpack config. Have you not corrected this?

Reply:
It is on #2 .